### ISSUE TITLE
Less -w prints carriage return ^M in blank lines

### ISSUE THREAD

**Issue by imbrish** [gwsw/less#474]
I have noticed a potential bug in less.

When viewing files with windows line endings (CRLF), `less -w` shows carriage return `^M` in blank lines. Lines which are not empty display correctly, skipping the carriage returns. Without the `-w` option the `^M` are omitted in blank lines too.

The same happens for `-W`. Adding `-r` removes the markers but is a rather unacceptable workaround.

```sh
$ printf 'A\r\n\r\nB\r\n' | LESS= less -F
A

B

$ printf 'A\r\n\r\nB\r\n' | LESS= less -Fw
A
^M
B

$ printf 'A\r\n\r\nB\r\n' | LESS= less -Fwr
A

B

```

When text is long enough to scroll down and then back up, the `^M` disappear. So, these are only shown in the newly printed lines.

```sh
$ for ((i=0; i<40; i++)); do printf 'A\r\n\r\nB\r\n'; done | LESS= less -w
(lines scrolled back to)
A

B
(freshly printed lines)
A
^M
B

```

Tested with git 2.43.0.windows.1, bash 5.2.21(1), less 643.

**Comment by gwsw** [comment]
Note: this bug was introduced in 1aad5cb07eb4f00deccdc6596c2d727be5702be2.

**Comment by gwsw** [comment]
Fixed in 94f1ae3618d93dc0c092a9a021c079b5db7e3831.
